spec/datadog/core/environment/cgroup_spec.rb

```diff
@@ -10,11 +10,11 @@
 
     context 'when the \'/proc/self/cgroup\' file is not present' do
       before do
+        expect(Datadog.logger).to_not receive(:error)
```

Comment:
I assume that calling `Datadog.logger` is how we make sure we initialise the logger?

Reply:
Yeah -- it's being initialized as a side-effect.

The current approach is not great (we should be able to have more control over using only a part of the library), but my intention here was just to stop the bleeding vs improving the approach here :)